apps/vor/pkg/postgres/postgres.go

```diff
@@ -235,3 +237,17 @@ type Weekday struct {
 	Day     time.Weekday `pg:",pk,use_zero"`
 	Class   Class
 }
+
+type LessonPlan struct {
+	Id          string `pg:"type:uuid"`
+	Title       string
+	Description string
+	ClassId     string `pg:"type:uuid"`
+	Repetition  int
+}
+
+type Attachment struct {
+	Id       string `pg:"type:uuid"`
+	FileName string
+	FilePath string
```

Comment:
Would it be better if we derive filename from filePath in the backend code rather than saving it in the DB? Just to reduce data duplication